youtube_dl/extractor/svt.py

```diff
@@ -37,6 +37,12 @@ def _extract_video(self, url, video_id):
                 })
         self._sort_formats(formats)
 
+        # SVT does not tell us the language, so we assume swedish.
+        subtitles = {}
+        for sr in video_info['subtitleReferences']:
+            if 'url' in sr:
+                subtitles.setdefault('sv', []).append({'url': sr['url']})
```

Comment:
yes but the language code (ISO-639) for swedish is sv :) https://en.wikipedia.org/wiki/Swedish_language